spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/testing.adoc

```diff
@@ -770,6 +779,15 @@ If you require more control over Spring REST Docs configuration than offered by
 include::{docs-java}/features/testing/springbootapplications/autoconfiguredspringrestdocs/withrestassured/MyRestDocsConfiguration.java[]
 ----
 
+If you want to make use of Spring REST Docs support for a parameterized output directory, you can create a `RestDocumentationFilter` bean.
+The auto-configuration calls `addFilter` with this filter, thereby causing each `RestAssured` call to automatically generate the default snippets.
```

Comment:
This isn't true, unfortunately. There's no auto-configuration for REST Assured in Spring Boot at the moment. Can you please revert the REST Assured part of these changes?

Reply:
ok!